custom/icds/models.py

```diff
@@ -119,9 +119,9 @@ def utility(self):
     def domain(self):
         return self.link.domain
 
-    def to_json(self, app_names):
+    def to_json(self):
         from custom.icds.serializers import HostedCCZSerializer
-        return HostedCCZSerializer(self, context={'app_names': app_names}).data
+        return HostedCCZSerializer(self).data
```

Comment:
I don't think getting the build doc is avoidable, because the tag is stored in the app's profile which isn't available in many places - this is also why I got rid of `get_brief_apps_in_domain`, the briefs don't have profile info.

I'll take a look at adding pagination.

Reply:
okay. If you can add pagination that would be super

Reply:
(non blocker)

Reply:
Added pagination. Also made search work via ajax instead of page reload.